client/src/index.jsx

```diff
@@ -63,26 +72,82 @@ class Header extends React.Component {
 		});
 	}
 
+	showRatings(e) {
+		e.preventDefault();
+		let show = (this.state.showDetails + 1)%2;
+		this.setState({
+			showDetails: show
+		});
+	}
+
+	close(e){
+		e.preventDefault();
+		this.setState({
+			showDetails: 0
+		});
+	}
+
 	render() {
-		return (<div>
+		return (<div id="header">
```

Comment:
Looks good, might be good to split this big render message into separate components for ease in testing, but not necessary.

Reply:
Yeap will do!
I'll work on testing tmr too.